aidial_adapter_vertexai/deployments.py

```diff
@@ -14,6 +14,7 @@ class ChatCompletionDeployment(str, Enum):
     GEMINI_PRO_1 = "gemini-pro"
     GEMINI_PRO_VISION_1 = "gemini-pro-vision"
     GEMINI_PRO_1_5 = "gemini-1.5-pro-preview-0409"
+    GEMINI_FLASH_1_5 = "gemini-1.5-flash"
```

Comment:
Not sure, what version to add here for prod

In docs there is three options:

- Latest version: `gemini-1.5-flash-latest`
- Latest stable version: `gemini-1.5-flash`
- Stable version: `gemini-1.5-flash-001`

Reply:
Let's use the stable version `gemini-1.5-flash-001`.

Reply:
Done